opendevin/server/listen.py

```diff
@@ -28,3 +28,10 @@ async def get_litellm_models():
     Get all models supported by LiteLLM.
     """
     return litellm.model_list
+
+@app.get("/litellm-agents")
+async def get_litellm_models():
+    """
+    Get all agents supported by LiteLLM.
+    """
+    return ["LangchainsAgent", "CodeActAgent", "PlannerAgent"]
```

Comment:
This PR is a great first step, but I'd love to be able to get the list dynamically. It should be available as `Agent._registry` (though we might want to create an `Agent.listAgents()` function instead of using that directly)

Reply:
Have added the functionality to get the list of agents dynamically

Reply:
added the function listAgents as well

Reply:
@rbren please have a look

Reply:
Looks great, thank you!